File: src/main/kotlin/OutputFormat.java
import java.util.*;

public class OutputFormat {
    public static void main(String[] args) {


        var list = new ArrayList<>();

        Scanner sc = new Scanner(System.in);
        System.out.println("================================");
        for (int i = 0; i < 3; i++) {
            String s1 = sc.next();
            int x = sc.nextInt();
            System.out.printf("%s", s1);
            for (int j = 0; j < 15 - s1.length(); j++) {
                System.out.printf(" ");
            }
            if (x > 9 && x < 100)
                System.out.printf("%s\n", "0" + x);
            else if (x >= 0 && x < 10)
                System.out.printf("%s\n", "00" + x);
            else
                System.out.printf("%d\n", x);
        }
        System.out.println("================================");
    }
}

File: src/main/kotlin/OutPutFormat.kt
import java.util.*

class OutPutFormatKotlin {

    fun show() {
        try {


            for (i in 0..3) {
                var scanner = Scanner(System.`in`)
                val s1 = scanner.next()
                val x1 = scanner.nextInt()
                print(s1)
                for (i in 0..15 - s1.length) {
                    print(" ")
                }
                println(x1)
            }
        }catch (e:Exception){}
    }

}